app/views/incomeTaxComparison/IncomeSummary.scala.html

```diff
@@ -41,16 +41,16 @@ <h2 id="incomeSummaryHeading" class="heading-large">@{incomeSummaryHeading}</h2>
         @messages("tai.incomeTaxComparison.incomeTax.fromEmployer.description")
     </p>
 
-    <table class="table--font-reset flush--bottom responsive" id="incomeSummaryComparisonTable" role="grid">
+    <table class="table--font-reset flush--bottom responsive" id="incomeSummaryComparisonTable" role="table">
```

Comment:
Was a role of grid appropriate for this? Was the table interactive? If not do you need a role of table?